index.html

```diff
@@ -1,49 +1,53 @@
-
 <!DOCTYPE html>
 <html lang='en'>
+
 <head>
-    <title>Media Department | Cornerstone Church </title>
-    <meta name='viewport' content='initial-scale=1.0'>
-    <meta name="apple-mobile-web-app-capable" content="yes">
+	<title>Media Department | Cornerstone Church </title>
+	<meta name='viewport' content='initial-scale=1.0'>
+	<meta name="apple-mobile-web-app-capable" content="yes">
 	<link rel="apple-touch-icon" href="touch-icon-iphone.png">
 	<link rel="apple-touch-icon" sizes="152x152" href="touch-icon-ipad.png">
 	<link rel="apple-touch-icon" sizes="180x180" href="touch-icon-iphone-retina.png">
 	<link rel="apple-touch-icon" sizes="167x167" href="touch-icon-ipad-retina.png">
```

Comment:
LINES: 8-11

What are these for? Are they needed?

Reply:
Transferred from the template I made on other sites. Used for fav icons on different devices.

Reply:
Alright can we remove them?

Reply:
Do we have a status on this pull request?